app/views/contexts/preview/preview_question.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Union
+
+from flask_babel import lazy_gettext
+from werkzeug.datastructures import ImmutableDict
+
+
+class PreviewQuestion:
+    def __init__(
+        self,
+        *,
+        block: ImmutableDict,
+    ):
+        self._block = block
+        self._block_id = block["id"]
+        self._question = self.get_question()
+        self._title = self._question["title"]
+        self._answers = self._build_answers()
+        self._descriptions = self._question.get("description")
+        self._guidance = self._question.get("guidance")
+        self._type = self._question.get("type")
+
+    def _build_answers(self) -> list[dict]:
+        answers_list = []
+        for answer in self._question.get("answers", []):
+            answer_dict = {}
+            answer_type = answer.get("type")
+            if (answer_label := answer.get("label")) and (answer_type != "Checkbox"):
+                answer_dict["label"] = answer_label
+
+            if options := answer.get("options"):
+                options_list = [option["label"] for option in options]
+                answer_dict["options"] = options_list
+                if answer_type == "Radio":
+                    answer_dict["options_text"] = lazy_gettext(
+                        "You can answer with one of the following options:"
+                    )
+                else:
+                    answer_dict["options_text"] = lazy_gettext(
+                        "You can answer with the following options:"
+                    )
```

Comment:
Should be if Checkbox else. As Dropdown is same as radio

Reply:
Added.